tracer/build/_build/NukeExtensions/DotNetSettingsExtensions.cs

```diff
@@ -91,6 +91,18 @@ public static T EnableNoDependencies<T>(this T settings)
         return settings.SetProperty("BuildProjectReferences", false);
     }
 
+    public static DotNetTestSettings EnableCrashDumps(this DotNetTestSettings settings, MiniDumpType dumpType = MiniDumpType.MiniDumpWithPrivateReadWriteMemory)
+    {
+        if (bool.Parse(Environment.GetEnvironmentVariable("enable_crash_dumps") ?? "false"))
+        {
+            return settings
+                .SetProcessEnvironmentVariable("COMPlus_DbgEnableMiniDump", "1")
+                .SetProcessEnvironmentVariable("COMPlus_DbgMiniDumpType", ((int) dumpType).ToString());
+        }
+
+        return settings;
+    }
```

Comment:
Maybe not for now, but to ensure that we will capture process crashes as dumps whenever possible we might want to consider enabling the [Windows User-Mode Dump collection](https://learn.microsoft.com/en-us/windows/win32/wer/collecting-user-mode-dumps) (and on Linux playing with `ulimit`?)
The .NET Dump capturer is good but not perfect, I have experienced situations where the Windows Dump collector succeeded while the .NET one failed.
Just something to consider.